web/app/js/components/BaseTable.jsx

```diff
@@ -89,8 +90,13 @@ class BaseTable extends React.Component {
   generateRows = (tableRows, tableColumns, order, orderBy, filterBy) => {
     let rows = tableRows;
     let col = _find(tableColumns, d => d.dataIndex === orderBy);
+    let {defaultOrderBy} = this.props;
+    if (_isEmpty(orderBy) && !_isEmpty(defaultOrderBy)) {
```

Comment:
is there an advantage to using `if (_isEmpty(orderBy) && !_isEmpty(defaultOrderBy))` instead of `if (!orderBy && defaultOrderBy)` here? 

since ‘orderBy’ and ‘defaultOrderBy’ are always strings and would never contain boolean values, do we need lodash? (and if there is an advantage, should we be using this in line 98 as well)?

Reply:
I realized that `orderBy` is being set by `defaultOrderBy` in the constructor, and using `_orderBy` was enough to fix this bug, so I'm removing this check :)

but you're right, we don't need the isEmpty here if those are always strings!